File: src/main/java/com/david/Hello.java
package com.david;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello Java");
        Person p = new Person("David",66.5f,1.7f);
        p.hello();
        p.weight = 66.5f;
        p.height = 1.7f;
        System.out.println(p.bmi());

        /*int age = 19;
        Integer age2 = 19;
        char c = 'A';
        Character c2 = 'B';
        byte b = 120;
        float weight = 66.5f;
        float height = 1.7f;
        boolean adult = true;
        boolean enroll = false;*/

        String name = "David";
    }
}

File: src/main/kotlin/com/david/Hello.kt
package com.david

import javafx.scene.text.FontWeight

fun main(args: Array<String>) {
//    println("Hello Kotlin")
//     Human().hello()
    val h = Human(weight = 66.5f,height = 1.7f)
    println(h.bmi())

    /*var age = 19
    age = 20
    var weight = 66.5f
    var name : String
    name = "David"*/

}
class Human(var name:String = "",var weight:Float , var height:Float ){
    init {
        println("test$weight")
    }

//    constructor(name:String,weight: Float,height: Float):this(weight,height)

    fun bmi():Float{
        val bmi = weight /(height*height)
        return  bmi
    }

    fun hello(){
        println("Hello Kotlin")
    }
}